#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_check-for-octave2.9.dpatch by Rafael Laboissiere <rafael@debian.org>
##
## DP: Abort toggle_plplot_use if not using Octave 2.9 or newer

@DPATCH@

--- plplot-5.6.0.rc1.5.6.1.orig/bindings/octave/PLplot/toggle_plplot_use.m
+++ plplot-5.6.0.rc1.5.6.1/bindings/octave/PLplot/toggle_plplot_use.m
@@ -20,6 +20,11 @@
 
 1;
 
+## Abort if not using Octave 2.9 or newer
+if ! ((split (version , '.') (1, 1) == '2' && split (version , '.') (1, 2) == '9') || split (version , '.') (1, 1) > '2' ) 
+  error ("This version of octave-plplot only works with Octave 2.9 or newer\n");
+endif
+
 if ! exist ("use_plplot_state")
   global use_plplot_state
   use_plplot_state = "on";
